x-pack/plugins/reporting/server/usage/get_reporting_usage.ts

```diff
@@ -0,0 +1,159 @@
+  if (statusBuckets) {
+    statusTypes = getKeyCount(statusBuckets) as {
+      completed: number;
+      failed: number;
+    };
+  }
+
+  return { _all: all, status: statusTypes, ...jobTypes };
+}
+
+type RangeStatSets = Partial<
+  RangeStats & {
+    lastDay: RangeStats;
+    last7Days: RangeStats;
+  }
+>;
+async function handleResponse(server: any, response: AggregationResults): Promise<RangeStatSets> {
+  const buckets = get(response, 'aggregations.ranges.buckets');
+  if (!buckets) {
+    return {};
+  }
+  const { lastDay, last7Days, all } = buckets as RangeAggregationResults;
+
+  const lastDayUsage = lastDay ? getAggStats(lastDay) : ({} as RangeStats);
+  const last7DaysUsage = last7Days ? getAggStats(last7Days) : ({} as RangeStats);
+  const allUsage = all ? getAggStats(all) : ({} as RangeStats);
+
+  return {
+    last7Days: last7DaysUsage,
+    lastDay: lastDayUsage,
+    ...allUsage,
+  };
+}
+
+export async function getReportingUsage(server: any, callCluster: any) {
+  const config = server.config();
+  const reportingIndex = config.get('xpack.reporting.index');
+
+  const params = {
+    rest_total_hits_as_int: true,
+    index: `${reportingIndex}-*`,
+    filterPath: 'aggregations.*.buckets',
+    body: {
+      size: 0,
+      aggs: {
+        ranges: {
+          filters: {
+            filters: {
+              all: { match_all: {} },
+              lastDay: { range: { created_at: { gte: 'now-1d/d' } } },
+              last7Days: { range: { created_at: { gte: 'now-7d/d' } } },
+            },
+          },
+          aggs: {
+            [JOB_TYPES_KEY]: { terms: { field: JOB_TYPES_FIELD, size: DEFAULT_TERMS_SIZE } },
+            [STATUS_TYPES_KEY]: { terms: { field: STATUS_TYPES_FIELD, size: DEFAULT_TERMS_SIZE } },
+            [OBJECT_TYPES_KEY]: {
+              filter: { term: { jobtype: PRINTABLE_PDF_JOBTYPE } },
+              aggs: { pdf: { terms: { field: OBJECT_TYPES_FIELD, size: DEFAULT_TERMS_SIZE } } },
+            },
+            [LAYOUT_TYPES_KEY]: {
+              filter: { term: { jobtype: PRINTABLE_PDF_JOBTYPE } },
+              aggs: { pdf: { terms: { field: LAYOUT_TYPES_FIELD, size: DEFAULT_TERMS_SIZE } } },
+            },
+          },
+        },
+      },
+    },
+  };
+
+  return callCluster('search', params)
+    .then((response: AggregationResults) => handleResponse(server, response))
+    .then(async (usage: UsageObject) => {
+      // Allow this to explicitly throw an exception if/when this config is deprecated,
+      // because we shouldn't collect browserType in that case!
+      const browserType = config.get('xpack.reporting.capture.browser.type');
+
+      const xpackInfo = server.plugins.xpack_main.info;
+      const exportTypesHandler = await getExportTypesHandler(server);
+      const availability = exportTypesHandler.getAvailability(xpackInfo) as FeatureAvailabilityMap;
+
+      const { lastDay, last7Days, ...all } = usage;
+
+      return {
+        available: true,
```

Comment:
these are static?  is this info still useful?

Reply:
The top-level `available` flag is static, because the feature is available for all licenses (Basic license provides CSV export)

The field is here for compatibility with existing behavior. Removing it would also impact the `Chromium API > Usage` tests.